Exercise 5.24: Revise your program to throw an exception if the second
number is zero. Test your program with a zero input to see what happens on
your system if you donâ€™t catch an exception.

Answer:
int first,sec;
std::cin >> first>>sec;
if(sec == 0)
{
  throw runtime_error("Division by zero);
}
return first/sec;
